examples/all/manifests/thanos-receive-statefulSet.yaml

```diff
@@ -38,6 +38,17 @@ spec:
               - thanos
               topologyKey: kubernetes.io/hostname
             weight: 100
+          - podAffinityTerm:
```

Comment:
Hmm I would expect podAntiAffinity, no?

Reply:
The wording is extremely confusing I agree, but it's under the `podAntiAffinity` object, so this does what we expect.

Reply:
ah, diff was not even showing this word... (: Thanks